certbot-dns-linode/certbot_dns_linode/__init__.py

```diff
@@ -0,0 +1,86 @@
+"""
+The `~certbot_dns_linode.dns_linode` plugin automates the process of
+completing a ``dns-01`` challenge (`~acme.challenges.DNS01`) by creating, and
+subsequently removing, TXT records using the Linode API.
+
+
+Named Arguments
+---------------
+
+==========================================  ===================================
+``--dns-linode-credentials``                Linode credentials_ INI file.
+                                            (Required)
+``--dns-linode-propagation-seconds``        The number of seconds to wait for
+                                            DNS to propagate before asking the
+                                            ACME server to verify the DNS
+                                            record.
+                                            (Default: 930)
```

Comment:
Thanks for updating the default value below. I believe this should be updated to match.

https://github.com/certbot/certbot/blob/96aeb0b5479d869d9d6176f1e033ef88ddd41bdd/certbot-dns-linode/certbot_dns_linode/dns_linode.py#L32